.gitignore

```diff
@@ -1,48 +1,22 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
 #
 # If you find yourself ignoring temporary files generated by your text editor
 # or operating system, you probably want to add a global ignore instead:
-#   git config --global core.excludesfile ~/.gitignore_global
+#   git config --global core.excludesfile '~/.gitignore_global'
 
-# Ignore bundler config
+# Ignore bundler config.
 /.bundle
 
-# Ignore the default SQLite database.
-/db/*.sqlite3
-/db/*.sql
-
 # Ignore all logfiles and tempfiles.
-/log/*.log
-/tmp
-
-# OS X Files
-.DS_Store
-
-# Ignore coveralls results
-coverage/*
-/coverage/*
-public/assets/source_maps/
-
-# Ignore coveralls results
-coverage/*
-/coverage/*
-
-# Images
-public/uploads
-
-dump.rdb
-
-# Ignore debugging history
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+# Ignore uploaded files in development.
+/storage/*
+!/storage/.keep
 .byebug_history
 
```

Comment:
Add `/test` folde since we are not gonna use Minitest, we are going to use Rspec instead

Reply:
I don't mind adding the /test folder in the .gitignore but why do you want to add it? I think it would be better not to have that folder and that will be enough.

Reply:
The reason to add the `/test` folder in the `.gitignore` is to allow people to use the scafolding functionality of rails. If we don't add the `/test` folder, people who used the scafolding may think is tedious to remove the `/test` folder everytime it's being used

Reply:
I have done something better: remove entirely the support of Test::Unit on the Rails app (with -T modifier on the rails new command), so now Rails would not generate a test folder nor any files of Test::Unit when scaffolding models. So, it won't be necessary to add the test folder to this file.